tools/cassandra/handler.go

```diff
@@ -150,11 +149,28 @@ func createKeyspace(cli *cli.Context) error {
 	}
 	err = doCreateKeyspace(*config, keyspace)
 	if err != nil {
-		return handleErr(fmt.Errorf("error creating Keyspace:%v", err))
+		return handleErr(fmt.Errorf("error creating Keyspace:%+v", err))
 	}
 	return nil
 }
 
+func checkHealth(cli *cli.Context) error {
+	config, err := newCQLClientConfig(cli)
+	if err != nil {
+		return handleErr(schema.NewConfigError(err.Error()))
+	}
+
+	config.Keyspace = systemKeyspace
+
+	client, err := newCQLClient(config)
+	if err != nil {
+		return handleErr(fmt.Errorf("unable to establish CQL session:%+v", err))
+	}
+
+	defer client.Close()
```

Comment:
i guess we don't even need to `defer`, but this is good regression-proofing. ; )